src/styles/_workspace.scss

```diff
@@ -563,6 +563,12 @@ $code-color-error: #ff4444;
         }
       }
     }
+
+    .sa-html-display {
+      background-color: white;
+      width: 100%;
+      height: 60vh;
```

Comment:
There's an unnecessary scrollbar in the mobile view that can be wiggled about.

![vmconnect_xpU3TMehtD](https://user-images.githubusercontent.com/5585517/194616597-d551b84e-66b6-4141-a3d3-ed41d47f64b5.gif)